css/style.css

```diff
@@ -0,0 +1,239 @@
+    gap: .8rem;
+}
+.humbager .logo i{
+    color: red;
+    font-size: 2rem;
+}
+.humbager .logo h1{
+    font-stretch:expanded;
+    font-size: 1.2rem;
+}
+
+.search-nav{
+    position: relative;
+    width: 40%;
+    margin: 0 auto;
+    padding: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    column-gap: 3%;
+
+}
+.search{
+    display: flex;
+    background-color: rgb(232, 232, 236);
+    width: 100%;
+    border-radius: 1.8rem;
+    overflow: hidden;
+    padding: 0 1rem;
+
+}
+.search input[type='search']{
+    width: 93%;
+    font-size: 1.1rem;
+    background-color: transparent;
+
+}
+.search input[type='search']::placeholder{
+    padding: 0 1rem;
+    font-size: 1.1rem;
+}
+.search i{
+    display: flex;
+    justify-content: center;
+    color:black;
+    width: 100%;
+    padding: 0.2rem;
+    font-size: 1.4rem;
+    cursor: pointer;
+}
+.search i:hover{
+    transform: scale(.8);
+    opacity: 0.8;
+    transition: 0.3s ease;
+}
+
+#microphone, .profile i{
+    cursor: pointer;
+    font-size: 1.5rem;
+}
+#microphone:hover,.humbager i:hover,.profile i:hover{
+    transform: scale(0.8);
+}
+
+.profile{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    column-gap: 1.2rem;
+}
+/* End of navigation bar */
+
+main{
+    width: 100%;
+}
+
+aside{
+    max-width: 25%;
+    padding: 1rem 0;
+
+}
+.nav-menu{
+    display: flex;
+    flex-direction: column;
+
+}
+.nav-item {
+    display: grid;
+    padding: 1rem;
+    place-content: start;
+}
+.nav-item:hover{
+    background-color: rgb(232,235,240);
+    cursor: pointer;
+    transform: scale(.9);
+    border-radius: 1.2rem;
+    transition: .3s ease-in-out;
+}
+.nav-item i{
+    font-size: 1rem;
+}
+.nav-item a{
+    font-size: .6rem;
+}
+
+/* videos section */
+
+.video{
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 2rem;
+    flex-wrap: wrap;
+    margin: 0 1rem;
+ }
+ .video .card{
+    width: 22%;
+    border-radius: 1rem;
+    display: block;
+    padding-block: 1rem;
+
+ }
+ .video-section{
+    width: 100%;
+    aspect-ratio: 1/1;
+    border-radius: 1rem;
+    overflow: hidden;
+ }
+ .video-section img{
+    width: 100%;
+
+ }
+ .title{
+    display: flex;
+    padding: .6rem 0;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+ }
+
+ .description>p{
+    font-weight: bold;
+ }
+ .person-name{
+   font-size: .8rem;
+   padding-top: 0.3rem;
+ }
+.person-name p{
+    line-height: 1.3;
+}
+.active{
+    display: block;
+}
+@media screen and (max-width: 767px) {
+    .video{
+        margin: 0 0.2rem;
+    }
+    .video .card{
+        width: 50%;
+
+    }
+    aside{
+        display: none;
+    }
+}
+@media screen and (max-width: 992px) {
+    .video .card{
+        width: 30%;
+    }
+}
+@media screen and (max-width: 600px) {
+    .video .card{
+        width: 80%;
+    }
+    aside{
+        display: none;
+    }
+}
```

Comment:
- [Optional] Kindly, you already told him to set up the linter, and he will fix those things when he tries to fix the linter error, I think adding this comment is not necessary here, please remove it👌
- [Optional] Great job for adding the comment politely and with reasons 👍,  I think it would be better if you use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.